resources/core/charts/ui-api/templates/deployment.yaml

```diff
@@ -18,15 +18,17 @@ spec:
       maxUnavailable: 0
   template:
     metadata:
-      annotations:
-        sidecar.istio.io/inject: "true"
       labels:
         app: {{ template "name" . }}
         chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
         release: "{{ .Release.Name }}"
         heritage: "{{ .Release.Service }}"
     spec:
       serviceAccountName: {{ template "fullname" . }}
+      {{ if .Values.global.isLocalEnv }}
+      hostNetwork: true
```

Comment:
Do we need that?

Reply:
Yes. `dex.kyma.local` is not resolvable by DNS as there is static mapping in hosts `/etc/hosts`. Because of that component is not able to call dex and thus initialise OIDC authenticator.